doc/tutorials/basic/develop_notebook.md

```diff
@@ -0,0 +1,357 @@
+# Develop in a Notebook
+
+In this section, we will learn the basics of developing Panel apps efficiently in a notebook:
+
+- Preview a notebook app in Jupyter lab with the *Jupyter Panel Preview*.
+- Serve a notebook app with autoreload using `panel serve app.ipynb --autoreload`.
+- Inspect a component using `SHIFT+Tab` and `print`.
+- Inspect a component's class and instance parameters using `.param`.
+
+:::{note}
+When we are asked to *run the code* in the sections below, we may either execute the code directly in the Panel docs via the green *run* button, in a cell in a notebook, or in a file `app.ipynb` that is served with `panel serve app.ipynb --autoreload`.
+:::
+
+## Install the Dependencies
+
+Please make sure [Matplotlib](https://matplotlib.org/) and [Numpy](https://numpy.org/) are installed.
+
+::::{tab-set}
+
+:::{tab-item} conda
+:sync: conda
+
+``` bash
+conda install -y -c conda-forge matplotlib numpy
+```
+:::
+
+:::{tab-item} pip
+:sync: pip
+
+``` bash
+pip install matplotlib numpy
+```
+:::
+
+::::
+
+## Preview your app
+
+A simple notebook file containing a Panel app could look like.
+
+![Panel Notebook Example App](../../_static/images/develop_notebook_simple_example.png)
+
+Start from an empty notebook named `app.ipynb`.
+
+Copy-paste the 3 code cells below into the notebook
+
+```python
+import panel as pn
+
+pn.extension()
+```
+
+```python
+pn.panel("Hello World").servable()
+```
+
+```python
+pn.panel("Hello Again")
+```
+
+Run the cells and save it as `app.ipynb` if you have not already done it.
+
+It should look like
+
+![Panel Notebook App](../../_static/images/develop_notebook_simple_example.png)
+
+:::{note}
+The code in the notebook refers to
+
+- `panel`: The Panel python package. It's a convention to import it as `pn`.
+- `pn.extension()`: **Loads the [`pyviz_comms`](https://github.com/holoviz/pyviz_comms) notebook extension**, loads javascript dependencies, and configures Panel.
+- `pn.panel(...)`: Creates a *displayable* Panel component. **The component can be displayed directly in the notebook**.
+- `.servable()`: Displays the component in a *server app*.
+:::
```

Comment:
Redundant; can be linked back to the basics if necessary.

Reply:
Resolved by Tabs. Feel free to reopen if you do not think so.